python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -206,6 +206,23 @@ def get_summary_text(self):
         """
         return self._get_summary_text()
 
+    def get_combined_model(self):
+        """
+        Return single-GPU model for serialization.
+
+        Returns
+        -------
+
+        model : Trained single-GPU model or None if the model has not
+               yet been trained.
+        """
+
+        # set internal model if it hasn't been accessed before
```

Comment:
Can we move these to dask.ensemble.base instead of the two versions?

Reply:
Oh sure, I can try that. I didn't think that would work because `cuml.dask.ensemble.base.BaseRandomForestModel` inherits from `object` and doesn't have `self._get_internal_model()` defined.

But I see now that it references that method (https://github.com/rapidsai/cuml/blob/816bb65069e2b8fd0e02be5d01eb62c5c153665a/python/cuml/dask/ensemble/base.py#L172), so I guess that class isn't intended to be used by itself and assumes it's only used as a mixin together with `cuml.dask.common.base.BaseEstimator`

Reply:
moved in https://github.com/rapidsai/cuml/pull/3388/commits/83f10b73ed5139053d6261cbeb729a60b1980836